### ISSUE TITLE
[Content] Importer and processor doesn't show up in the Content Pipeline tool using MonoGame 3.7

### ISSUE THREAD

**Issue by megabobik** [craftworkgames/MonoGame.Extended#557]
Hi,

recently new version of MonoGame was released, will ME be updated soon, or it need some rework to adapt new version?

**Comment by megabobik** [comment]
https://github.com/craftworkgames/MonoGame.Extended/pull/558

**Comment by craftworkgames** [comment]
Just to be clear, I've renamed this issue to better frame what problem we're trying to solve here.

> After adding a reference to MonoGame.Extended.Content.Pipeline.dll the importer and processor doesn't show up in the Content Pipeline tool using MonoGame 3.7

At this stage I'm not convinced that updating MonoGame.Extended to use MonoGame 3.7 will actually solve the above problem. As far as I can tell this is actually a bug release in MonoGame. It's still a problem for us though, so I'll leave this issue open. Hopefully we can come up with a solution or workaround in the meantime.

**Comment by lofcz** [comment]
Any update on this one? Rendering `MonoGame.Extended.BitmapFonts` unusable.

**Comment by craftworkgames** [comment]
So I had a chance to look into this a few days ago but only just now I've found time to write up something about it.

Here's what I did:

1. I pulled the latest version of MonoGame.Extended onto a machine that had MonoGame 3.7 installed.
2. I built and run the demos without any issues.
3. I opened the MonoGame Content Pipeline tool and confirmed that indeed the Importer and Processor show up as missing in the GUI and can't be selected (see screenshot).
4. However, there are no issues actually building the content (see screenshot).

![image](https://user-images.githubusercontent.com/3201643/49152284-0cadeb00-f35e-11e8-896c-d0fabdd2e95c.png)

Next I downloaded and installed the [latest development build of MonoGame](http://www.monogame.net/downloads/) which happens to be MonoGame 3.8.0.173

Repeating the same process I can confirm that this issue has been fixed in MonoGame and is not an issue in MonoGame.Extended.

![image](https://user-images.githubusercontent.com/3201643/49152735-7c70a580-f35f-11e8-956d-08e1b6a0bf5f.png)

So in conclusion there's essentially 2 workarounds to this problem.

1. You can install a development build of MonoGame 3.8.
2. If you need to continue using 3.7 for some reason, you can manually edit the `Content.mgcb` file to select the importer and processor. For example:

```
#begin small-font.fnt
/importer:BitmapFontImporter
/processor:BitmapFontProcessor
/build:small-font.fnt
```
